checker/designatorresolution/designator_resolution.go

```diff
@@ -1,8 +1,7 @@
-package phase
+package designatorresolution
```

Comment:
Add a simple package doc. See lexer, parser description in https://godoc.org/github.com/bazo-blockchain/lazo

Do it for all packages